logfire/_internal/config.py

```diff
@@ -156,6 +156,7 @@ def configure(
     fast_shutdown: bool = False,
     scrubbing_patterns: Sequence[str] | None = None,
     scrubbing_callback: ScrubCallback | None = None,
+    fstring_magic: bool | None = None,
```

Comment:
I think we said we'd change it for `fstring_inspect` or something? I don't recall the conversation properly.

Reply:
Changed to `inspect_arguments`